fail2ban/server/banmanager.py

```diff
@@ -26,6 +26,9 @@
 
 from threading import Lock
 
+import dns.exception
```

Comment:
well, still would be lovely to spit out somehow (stderr) and informative message to the user, and then yes embed error into "Banned ASN list"